plugin/trino-delta-lake/src/main/java/io/trino/plugin/deltalake/DeltaLakeAnalyzeProperties.java

```diff
@@ -30,15 +30,24 @@
 import java.util.Set;
 
 import static com.google.common.collect.ImmutableSet.toImmutableSet;
+import static io.trino.plugin.deltalake.DeltaLakeAnalyzeProperties.AnalyzeMode.NORMAL;
 import static io.trino.spi.StandardErrorCode.INVALID_ANALYZE_PROPERTY;
+import static io.trino.spi.session.PropertyMetadata.enumProperty;
 import static io.trino.spi.type.TimeZoneKey.UTC_KEY;
 import static io.trino.spi.type.VarcharType.VARCHAR;
 import static java.lang.String.format;
 
 public class DeltaLakeAnalyzeProperties
 {
+    enum AnalyzeMode
+    {
+        NORMAL,
```

Comment:
`NORMAL` sounds non-descriptive to me. I guess most people can't understand what "normal" is. 
How about removing this entry and using `null` instead? We can add a new entry when we come up with a better name.

Reply:
What about INCREMENTAL ? Would that work for you ?